src/j1cli.js

```diff
@@ -336,17 +338,42 @@ async function provisionRulePackAlerts(j1Client, rules, defaultSettings) {
   for (const r of rules) {
     if (r.instance) {
       const update = r.instance.id !== undefined;
-      promises.push(j1Client.mutateAlertRule(rule, update));
+      promises.push(j1Client.mutateAlertRule(r, update));
     }
     else {
-      const instance = {
-        name: r.name,
-        description: r.description,
-        question: {
-          queries: r.queries
-        },
-        ...defaultSettings
-      }
+      const instance = r.alertLevel ? 
+        {
+          name: r.name,
+          description: r.description,
+          question: {
+            queries: r.queries
+          },
+          specVersion: defaultSettings.specVersion,
+          pollingInterval: defaultSettings.pollingInterval,
+          outputs: defaultAlertSettings.outputs,
+          operations: [
+            {
+              when: defaultSettings.operations[0].when,
+              actions: [
+                {
+                  type: "SET_PROPERTY",
+                  targetProperty: "alertLevel",
+                  targetValue: r.alertLevel
+                },
+                {
+                  type: "CREATE_ALERT"
+                }
+              ]
+            }
+          ]
+        } : {
+          name: r.name,
+          description: r.description,
+          question: {
+            queries: r.queries
+          },
+          ...defaultAlertSettings
```

Comment:
This is a side comment, but spreading at the end of the object will actually override the previous instance of that property. I think it would make sense to actually spread this at the beginning of the object instead of the end. That way if we added any other properties to rule packs, they wouldn't get overwritten by the `defaultAlertSettings`.

```js
{
  ...defaultAlertSettings,
  name: r.name,
  description: r.description,
  question: {
    queries: r.queries
  }
}
```